internal/bminventory/inventory_test.go

```diff
@@ -1289,7 +1294,7 @@ var _ = Describe("GetNextSteps", func() {
 		var err error
 		expectedStepsReply := models.Steps{NextInstructionSeconds: defaultNextStepIn, Instructions: []*models.Step{{StepType: models.StepTypeInventory},
 			{StepType: models.StepTypeConnectivityCheck}}}
-		h1, err := common.GetHostFromDB(db, clusterId.String(), hostId.String())
+		h1, err := common.GetHostFromDB(db, infraEnvId.String(), hostId.String())
 		Expect(err).ToNot(HaveOccurred())
 		mockHostApi.EXPECT().GetNextSteps(gomock.Any(), gomock.Any()).Return(expectedStepsReply, err)
 		reply := bm.GetNextSteps(ctx, installer.GetNextStepsParams{
```

Comment:
There are already tests (copied) for V2GetNextSteps.  Why touch this one?

Reply:
In  this commit we have moved to using only one core function for accessing the hosts: GetHostFromDB. It look for the host based on id and infra-env.  The code was changed accordingly, and the test code also